sql/catalyst/src/main/java/org/apache/spark/sql/vectorized/ArrowColumnVector.java

```diff
@@ -19,19 +19,14 @@
 
 import org.apache.arrow.vector.*;
 import org.apache.arrow.vector.complex.*;
-import org.apache.arrow.vector.holders.NullableIntervalDayHolder;
 import org.apache.arrow.vector.holders.NullableVarCharHolder;
 
 import org.apache.spark.sql.util.ArrowUtils;
 import org.apache.spark.sql.types.*;
 import org.apache.spark.unsafe.types.UTF8String;
 
-import static org.apache.spark.sql.catalyst.util.DateTimeConstants.MICROS_PER_DAY;
-import static org.apache.spark.sql.catalyst.util.DateTimeConstants.MICROS_PER_MILLIS;
-
 /**
- * A column vector backed by Apache Arrow. Currently calendar interval type and map type are not
- * supported.
```

Comment:
Don't want to keep `map type ....`?

Reply:
Map type was added from https://github.com/apache/spark/pull/30393 but the fix was missing 😂